docs/getting-started/README.md

```diff
@@ -1 +1,29 @@
-TBA.
\ No newline at end of file
+# Fleet Getting Started Tutorials
+
+This documentation provides a number of tutorials which can help you get started with Fleet,
+the open-source multi-cluster management solution, developed by Microsoft Azure;
+through them, you will learn about the setup of Fleet and its core functionalities.
+
+> Note
+>
+> All tutorials listed assume that you already have a basic understanding of Kubernetes. For more
+> information, see the [Kubernetes documentation](https://kubernetes.io/docs/home/).
+
+Fleet can work with many different Kubernetes environment setups. You may run Fleet on managed
+Kubernetes clusters, such as
+[Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-us/products/kubernetes-service)
+or on on-premises Kubernetes clusters. It even works with local [kind](https://kind.sigs.k8s.io/)
```

Comment:
nit: I think on premises cluster is a much more attractive value proposition than kind.